go.mod

```diff
@@ -15,6 +15,7 @@ require (
 	github.com/mattn/go-colorable v0.1.7 // indirect
 	github.com/onsi/ginkgo v1.14.1
 	github.com/onsi/gomega v1.10.2
+	github.com/pkg/errors v0.9.1
```

Comment:
If you're indeed bloating the dependencies with this, at least squash this into the previous commit (the previous commit is broken since it doesn't have the module).

Reply:
I usually separate the auto generated stuff in a different commit to make the like of the reviewer easier :) 
As I understand the PR commits get squashed on approval anyway. 
but if you prefer I'll squash them :)

Reply:
> I usually separate the auto generated stuff in a different commit to make the life of the reviewer easier :)

Understandable.

> As I understand the PR commits get squashed on approval anyway.

They don't (in this repo).

> but if you prefer I'll squash them :)

Thanks.

Reply:
Done